### PR TITLE
Issue 1521: Updating TANF-app/doc/Background/

### LINKED ISSUE #1521: Update TANF-app/doc/Background/
### Context

After #1515 is merged, `TANF-app/doc/Background/` subdirectory markdowns need to be reviewed and updated. 

### ACs:
- [x] Add README.md w/ TOC
- [x] Update `Current-TDRS.md` (#1483)
- [x] Update any broken links within markdowns

### Tasks:
- [x] Create [hackmd](https://hackmd.io/@valeriecollignon/SyQ_qOE2F/edit) for subdir `README.md`  w/ TOC (`valcollignon` )
- [x] Create [hackmd](https://hackmd.io/6zK6oaMPQEOKgCsidMcUaA) to update `Current-TDRS.md` (`sreedevip` ,`adpennington`)
- [x] delete `TANF-app/docs/Background/Prototype/`
- [x] update Agency-Background.md with embedded image (`acf_org_chart.png`) from `TANF-app/docs/User-Experience/Design/design-assets/research-artifacts/` (which will be deleted with #1527)
- [x] rename `Flatfile Metadata Guide.md` to `2020, Fall - Flatfile Metadata Guide.md`  and move to `TANF-app/docs/User-Experience/Research/`
- [x] move `2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md` to `TANF-app/docs/User-Experience/Research/`
- [x] Open draft PR with subdirectory updates (`adpennington`)
- [x] Review and update any broken links (`valcollignon`, `adpennington`)
- [x] Create Figma diagram for data reporting process flow and add to `Current-TDRS.md` (`sreedevip`)

### Notes:
Part of the #1477 documentation update epic

### PR DESCRIPTION
**Pull request by ADPennington**
### Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1521 
_Acceptance criteria as stated in the issue_
- [x] Add [README.md w/ TOC](https://github.com/raft-tech/TANF-app/blob/99ffd69a7dffccb4c47f7b428453b83d59e8071a/docs/Background/README.md)
- [x] Update [Current-TDRS.md](https://github.com/raft-tech/TANF-app/blob/e449c87899cee61617c5c65041935db2fa4f0bf1/docs/Background/Current-TDRS.md) 
_Note: research-related links within this doc will be broken until this PR is merged_

As part of this work the following tasks were done:
- renamed and moved a couple of UX docs
- deleted `prototype` subdir
- updated `agency background.md` with org chart

### CODE REVIEW

File: docs/Background/Current-TDRS.md [hunk 1]
```diff
@@ -1,102 +1,271 @@
...
+- Section 2: 14kb - 2,000kb
+- Section 3: 1kb - 2kb
+- Section 4: 1kb - 2kb
+</details>
+
+**<details><summary>How does TDRS decrypt data files from STTs?</summary>**
+
+"Encryption" is an artifact of STTs using executables like FTANF.exe and SSPMOE.exe to prepare their data transmissions files. Encryption in this context means that the values in the SSN position of Section 1 and Section 2 data files are replaced with other alphanumeric characters. 
+    
+TDRS has a decryption script that runs on transmitted files if (and only if) the files have an "E" as the [encryption indicator in the header record](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research/2020,%20Fall%20-%20Flat%20File%20Metadata%20Guide.md#header-structure). For future reference: in the absence of checking the header record, if the SSN includes special characters, this is also evidence of encryption.
```
Comment (ADPennington) [inline docs/Background/Current-TDRS.md:167]:
```suggestion
TDRS has a decryption script that runs on transmitted files if (and only if) the files have an "E" as the [encryption indicator in the header record](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research/2020%2C%20Fall%20-%20Flat%20File%20Metadata%20Guide.md#header-structure). For future reference: in the absence of checking the header record, if the SSN includes special characters, this is also evidence of encryption.
```

File: docs/Background/Current-TDRS.md [hunk 2]
```diff
@@ -1,102 +1,271 @@
...
+
+**Important Note**: Acceptance and storage does not mean that transmitted data are error-free. OFA data staff conduct additional data quality checks to ensure accuracy, completeness, and consistency of the data within and across records and sections of the report over time. Data quality feedback reports are sent to STTs throughout the data reporting period (See feedback FAQ below for more details).
+</details>
+
+**<details><summary>What types of metadata does OFA collect about data transmissions?</summary>**
+
+TDRS tracks data transmissions from the point that data files are *successfully* picked up for processing (i.e. if files are not picked up, no metadata is tracked) to storage into the dB. Metadata collected include: when the data was processed, the reporting period, all information from the [header record](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research/2020,%20Fall%20-%20Flat%20File%20Metadata%20Guide.md#header-structure), which tells the system how to process and store the data, and a list of any errors detected by the system.
```
Comment (ADPennington) [inline docs/Background/Current-TDRS.md:193]:
```suggestion
TDRS tracks data transmissions from the point that data files are *successfully* picked up for processing (i.e. if files are not picked up, no metadata is tracked) to storage into the dB. Metadata collected include: when the data was processed, the reporting period, all information from the [header record](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research/2020%2C%20Fall%20-%20Flat%20File%20Metadata%20Guide.md#header-structure), which tells the system how to process and store the data, and a list of any errors detected by the system. 
```

File: docs/Background/Current-TDRS.md [hunk 3]
```diff
@@ -1,102 +1,271 @@
...
 
-The data is then uploaded using secure file transfer protocol (SFTP) into a system which then periodically attempts to process the data and import it into the database OFA staff uses for analysis. OFA staff access the data via direct read-only SQL queries using tools like python, Jupyter Notebooks, and SAS.
+Included herein is an overview of:
+- **[the data](#about-the-data)**, 
+- **[the legacy TDRS](#About-the-TANF-Data-Reporting-Sysyem-(TDRS))**, and
```
Comment (ADPennington) [inline docs/Background/Current-TDRS.md:7]:
```suggestion
- **[the legacy TDRS](#about-the-tanf-data-reporting-system-tdrs)**, and
```

File: docs/Background/Current-TDRS.md [hunk 4]
```diff
@@ -1,102 +1,271 @@
...
+### Data transmission
+
+**<details><summary>When are the data reports due?</summary>**
+
+Data reports are due within 45 days of the end of the fiscal quarter--
+
+| Fiscal Quarter | Calendar Period | Due Date    |
+| -------------- | --------------- | ----------- |
+| 1              | Oct 1 - Dec 31  | February 14 |
+| 2              | Jan 1 – Mar 3   | May 15      |
```
Comment (ADPennington) [inline docs/Background/Current-TDRS.md:63]:
```suggestion
| 2              | Jan 1 – Mar 31   | May 15      |
```

File: docs/Background/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+# Table of Contents
+* **[Agency Background](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Background/Agency-Background.md)** - ACF background and agency structure
+* **[TANF Program](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Background/TANF-Program.md)** - Background on the Temporary Assistance for Needy Families (TANF) program
+* **[Current TDRS](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Background/Current-TDRS.md)** - Background of the legacy TANF Data Reporting System (TDRS), including types of data states, territories and tribes (STTs) can submit, data preparation tools, OFA resources, communication, and report types
+* **[TDP Project Impact](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Background/Project-Impact.md)** - Business need, business impact, assumptions, and constraints
+* **[Acquisition](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Background/Acquisition/)** - Backgroud on the acquisition process, including the request for quotations (RFQ) and COR delegation letters
```
Comment (lfrohlich) [inline docs/Background/README.md:12]:
```suggestion
* **[Acquisition](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Background/Acquisition/)** - Background on the acquisition process, including the request for quotations (RFQ) and COR delegation letters
```

File: docs/Background/Current-TDRS.md [hunk 6]
```diff
@@ -1,102 +1,271 @@
...
-The TANF grantees usually generate their data in one of the following  ways (review [information about the incoming data format](https://www.acf.hhs.gov/ofa/resource/tanfedit/index#transmission-file-header)):
-* Using a legacy tool (ftanf.exe) that exports files in a special text format
-* Using their own software to export the data
-* Emailing their data to an OFA staff member to be entered for them
+OFA's TANF grantees in the states, territories, and tribes (STTs) are [legislatively-mandated](https://www.law.cornell.edu/uscode/text/42/611) to to report data via TDRS. OFA then aggregates the data and uses it for descriptive analyses and program performance monitoring, most notably through the [work participation rate calculations (WPR)](https://www.acf.hhs.gov/ofa/resource-library?f%5B0%5D=program%3A270&f%5B1%5D=program_topic%3A633&f%5B2%5D=type%3Aeasychart&keyword=&sort_by=combined_publication_date). The WPR measures how well states engage families receiving assistance in certain work activities during a fiscal year. STTs must meet both an overall (or “all families”) WPR and a two-parent work participation rate or be subject to a financial penalty.
```
Comment (lfrohlich) [inline docs/Background/Current-TDRS.md:3]:
```suggestion
OFA's TANF grantees in the states, territories, and tribes (STTs) are [legislatively-mandated](https://www.law.cornell.edu/uscode/text/42/611) to report data via TDRS. OFA then aggregates the data and uses it for descriptive analyses and program performance monitoring, most notably through the [work participation rate calculations (WPR)](https://www.acf.hhs.gov/ofa/resource-library?f%5B0%5D=program%3A270&f%5B1%5D=program_topic%3A633&f%5B2%5D=type%3Aeasychart&keyword=&sort_by=combined_publication_date). The WPR measures how well states engage families receiving assistance in certain work activities during a fiscal year. STTs must meet both an overall (or “all families”) WPR and a two-parent work participation rate or be subject to a financial penalty.
```

**Review by lfrohlich — CHANGES_REQUESTED** [review]
Looks good! Love the new expanded FAQ. Two minor edits noted

**Review by lfrohlich — APPROVED** [review]

**Review by valcollignon — APPROVED** [review]
Lgtm

**Pull request merged**